assets/templates/geography-homepage.tmpl

```diff
@@ -0,0 +1,25 @@
+<div class="slate background--gallery">
+    <div class="wrapper">
+        <nav class="breadcrumb">
+            <ol class="breadcrumb__list">
+                <li class="breadcrumb__item">
+                    <a class="breadcrumb__link" href="https://www.ons.gov.uk/">Home</a>
+                </li>
+                <li class="breadcrumb__item">
+                    Geography
+                </li>
+            </ol>
+        </nav>
+    <h1><b>{{.Metadata.Title}}</b></h1>
+    </div>
+
+</div>
+<div class="wrapper">
+    <h2 class="margin-bottom--2"><b>Area types</b></h2>
+    {{ range $i, $v := .Data.Items }}{{if $v.Name}}
+        <div class="background--gallery padding-top--2 padding-bottom--2 padding-right--1 padding-left--1 margin-bottom--2">
+            <h3 class="margin-top--0"><a class="area-type" id="area-type-{{$i}}" href="#">{{$v.Name}}</a></h3>
+        </div>
+    {{end}}{{end}}
+    <h2><b>Popular cities</b></h2>
```

Comment:
This redundant heading is still hanging about, it's might be worth removing until the 'Popular cities' functionality works?